src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -510,6 +509,9 @@ export class HaIntegrationCard extends LitElement {
           margin-top: 0;
           min-height: 24px;
         }
+        h3 {
+          word-wrap: break-word;
```

Comment:
Can we cap this at like 2-3 lines?

Reply:
`line-clamp`

Reply:
https://caniuse.com/css-line-clamp

Is good enough to use. We use it in the media browser as well.